packages/plugins/Traderxyz/src/Dashboard/index.tsx

```diff
@@ -0,0 +1,12 @@
+import type { Plugin } from '@masknet/plugin-infra'
```

Comment:
Looks like you're not using dashboard entry. Please remove those files.

Reply:
done.